books/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+from core.models  import TimeStampModel
+from users.models import User
+from stays.models import RoomOption
+
+class BookStatus(models.Model):
+    status = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'book_status'
```

Comment:
```suggestion
    class Meta:
        db_table = 'book_statuses'
```
class 명수는 단수로, 테이블명은 복수로 하는 것이 컨벤션입니다.

Reply:
확인 후 수정하겠습니다!